openfermioncirq/experiments/hfvqe/mfopt.py

```diff
@@ -0,0 +1,431 @@
+# pylint: disable=C
```

Comment:
What are we disabling?

Reply:
I'm ignoring the 80 character line limit in most places